pkg/alignchecker/alignchecker_test.go

```diff
@@ -119,3 +119,14 @@ func (t *AlignCheckerSuite) TestCheckStructAlignments(c *C) {
 		}
 	}
 }
+
+func (t *AlignCheckerSuite) TestCheckConsts(c *C) {
+	const foo = uint8(23)
+	const bar = uint64(12345678)
+	toCheck := map[string]reflect.Value{
+		"foo_const": reflect.ValueOf(foo),
+		"bar_const": reflect.ValueOf(bar),
+	}
+	err := valueCheck(path, toCheck)
+	c.Assert(err, IsNil)
```

Comment:
Can you add test cases for other conditions such as `toCheck` still having some elements left inside it and etc?

Reply:
Thanks. Added test cases, and also updated the code to iterate over `toCheck` elements, instead of having to delete them and check if  it the map is empty at the end.